src/prefect/server/database/orm_models.py

```diff
@@ -865,11 +865,6 @@ def job_variables(self):
         nullable=True,
         index=True,
     )
-
-    schedule = sa.Column(Pydantic(schemas.schedules.SCHEDULE_TYPES))
```

Comment:
We're dropping these from the ORM layer. In theory we've already migrated everyone to use `schedules` and `paused` back with the migration that added those. Should we drop these in a migration?